platform-api/che-core-api-git/src/main/java/org/eclipse/che/api/git/shared/RebaseRequest.java

```diff
@@ -0,0 +1,30 @@
+/*******************************************************************************
+ * Copyright (c) 2012-2015 Codenvy, S.A.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ *
+ * Contributors:
+ *   Codenvy, S.A. - initial API and implementation
+ *******************************************************************************/
+package org.eclipse.che.api.git.shared;
+
+import org.eclipse.che.dto.shared.DTO;
+
+/**
+ * Request to add rebase branch
+ *
+ * @author Dror Cohen
+ */
+@DTO
+public interface RebaseRequest {
+	/** set rebase operation */
+	void setOperation(String operation);
```

Comment:
I don't mean specific parameter. I just try to find better name. Something like branchName, but I don't like branchName because it can be branchName, commit hash etc. 
In git documentation https://git-scm.com/docs/git-rebase this parameter named just branch.
WDYT?